src/main/java/tr/com/orioninc/laborant/service/LabService.java

```diff
@@ -44,32 +49,34 @@ public static String connectAndExecuteCommand(String username, String password,
         }
         catch(Exception e)
         {
+            log.error("[connectAndExecuteCommand] Error occured while connecting to lab: {}" , e.getMessage());
             e.printStackTrace();
```

Comment:
Please log the stack trace as well.
as I remember log4j2 loggers can take exceptions as a parameter

Reply:
To be checked LabService.java line57